migrations/20200824194331-patient-gender.js

```diff
@@ -7,21 +7,39 @@ module.exports = {
         queryInterface.bulkUpdate(
           'patients',
           { gender: 'F' },
-          { gender: { [Sequelize.Op.regexp]: '.*' } },
-          { transaction: t }
-        ),
-      ]).then(() =>
-        queryInterface.changeColumn(
-          'patients',
-          'gender',
-          {
-            type: Sequelize.DataTypes.ENUM('M', 'F'),
-          },
           {
-            transaction: t,
+            gender: {
+              [Sequelize.Op.notIn]: ['F', 'Female', 'Male', 'M'],
+            },
```

Comment:
did you intentionally remove `transaction: t,` here?

Reply:
Good catch, should be fixed now